apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.tsx

```diff
@@ -237,45 +237,24 @@ const OperationD = (props: Props) => {
     : undefined;
   const editEthTx = useFeature("editEthTx");
   const editable = editEthTx?.enabled && isEditableOperation(mainAccount, operation);
-  // const dispatch = useDispatch();
-  // const handleOpenEditModal = useCallback(
-  //   (account, parentAccount, transactionRaw, transactionHash) => {
-  //     setDrawer(undefined);
-  //     if (subOperations.length > 0 && isToken) {
-  //       // if the operation is a token operation,(ERC-20 send), in order to speedup/cancel we need to find the subAccount
-  //       const opAccount = findSubAccountById(account, subOperations[0].accountId);
-  //       dispatch(
-  //         openModal("MODAL_EDIT_TRANSACTION", {
-  //           account: opAccount,
-  //           parentAccount: account,
-  //           transactionRaw,
-  //           transactionHash,
-  //         }),
-  //       );
-  //     } else {
-  //       dispatch(
-  //         openModal("MODAL_EDIT_TRANSACTION", {
-  //           account,
-  //           parentAccount,
-  //           transactionRaw,
-  //           transactionHash,
-  //         }),
-  //       );
-  //     }
-  //   },
-  //   [dispatch, isToken, subOperations],
-  // );
+  const dispatch = useDispatch();
 
-  const handleOpenEditModal = (
-    _account: unknown,
-    _parentAccount: unknown,
-    _transactionRaw: unknown,
-    _transactionHash: unknown,
-  ) => {};
+  const handleOpenEditModal = useCallback(() => {
+    setDrawer(undefined);
+
+    invariant(operation.transactionRaw, "operation.transactionRaw is required");
+
+    dispatch(
+      openModal("MODAL_EVM_EDIT_TRANSACTION", {
```

Comment:
Same here, as expected, in a more indirect way :slightly_smiling_face:

Reply:
Not as straightforward to fix.

Furthermore, this comes from the original PR (#2441) and did not seem to be challenged at the time (rightfully or not, I don't know)

cf. https://github.com/LedgerHQ/ledger-live/pull/2441/files#diff-42907b8096ef2aee623bba0ad42cb4f2a64bd09257203f8b8464e60a93331915R245-R277